org-code-javabuilder/lib/src/main/java/org/code/javabuilder/CodeBuilder.java

```diff
@@ -53,6 +60,12 @@ public void buildUserCode(List<String> compileList)
     this.compileCode(javaProjectFiles);
   }
 
+  public void buildUserAndValidationFiles() throws InternalServerError, UserInitiatedException {
```

Comment:
nit: mind adding a javadoc to differentiate this from `buildAllUserCode` and `buildUserCode`?